### ISSUE TITLE
App crashed when clicking on keyboard shortcuts

### ISSUE THREAD

**Issue by noteb00k** [laurent22/joplin#4049]
basically updating to the lastest version deleted one of my shortcuts so i clicked options, and in the options menu then I clicked keyboard shortcuts, and the application crashed.

I'll paste the message I got in in the end of this post

## Steps to reproduce

1.  I updated to version 1.3.15
2.  I toggled the note list with a shortcut I had personalized in a previous version (ctrl+space), and then from the options menu I toggled the sidebar. The view at the moment of the crash was sidebar + editor vew
3. I clicked options
4. I clicked Keyboard Shortcuts
5. it crashed

I tried several times and it keeps crashing... I think I'll downgrade until this is fixed.

## Describe what you expected to happen

Obviously I expected it to show me the keyboard shortcuts...

## Logfile

Error
Joplin encountered a fatal error and could not continue. To report the error, please copy the *entire content* of this page and post it on Joplin forum or GitHub.

Message
Command: toggleSidebar is unknown

Version info
Joplin for Desktop

Copyright © 2016-2020 Laurent Cozic
Joplin 1.3.15 (prod, win32)

Client ID: 973e9e3c6cc442dbb93493f905ee6a5a
Sync Version: 2
Profile Version: 34
Keychain Supported: No

Revision: 0cc544a95 (master)
Stack trace
Error: Command: toggleSidebar is unknown
    at Object.getLabel [as default] (C:\Users\%userprofile%\AppData\Local\Temp\1jpDq5r6O9blCMwLK5H26V58wCO\resources\app.asar\gui\KeymapConfig\utils\getLabel.js:32:19)
    at C:\Users\%userprofile%\AppData\Local\Temp\1jpDq5r6O9blCMwLK5H26V58wCO\resources\app.asar\gui\KeymapConfig\utils\useKeymap.js:21:34
    at Array.sort (<anonymous>)
    at allKeymapItems (C:\Users\%userprofile%\AppData\Local\Temp\1jpDq5r6O9blCMwLK5H26V58wCO\resources\app.asar\gui\KeymapConfig\utils\useKeymap.js:20:12)
    at C:\Users\%userprofile%\AppData\Local\Temp\1jpDq5r6O9blCMwLK5H26V58wCO\resources\app.asar\gui\KeymapConfig\utils\useKeymap.js:26:66
    at mountState (C:\Users\%userprofile%\AppData\Local\Temp\1jpDq5r6O9blCMwLK5H26V58wCO\resources\app.asar\node_modules\react-dom\cjs\react-dom.development.js:15476:20)
    at Object.useState (C:\Users\%userprofile%\AppData\Local\Temp\1jpDq5r6O9blCMwLK5H26V58wCO\resources\app.asar\node_modules\react-dom\cjs\react-dom.development.js:15918:16)
    at Object.useState (C:\Users\%userprofile%\AppData\Local\Temp\1jpDq5r6O9blCMwLK5H26V58wCO\resources\app.asar\node_modules\react\cjs\react.development.js:1562:21)
    at Object.useKeymap [as default] (C:\Users\%userprofile%\AppData\Local\Temp\1jpDq5r6O9blCMwLK5H26V58wCO\resources\app.asar\gui\KeymapConfig\utils\useKeymap.js:26:51)
    at exports.KeymapConfigScreen (C:\Users\%userprofile%\AppData\Local\Temp\1jpDq5r6O9blCMwLK5H26V58wCO\resources\app.asar\gui\KeymapConfig\KeymapConfigScreen.js:28:114)
Component stack

    in Unknown (created by ConfigScreenComponent)
    in div (created by ConfigScreenComponent)
    in div (created by ConfigScreenComponent)
    in div (created by ConfigScreenComponent)
    in ConfigScreenComponent (created by Connect(ConfigScreenComponent))
    in Connect(ConfigScreenComponent) (created by NavigatorComponent)
    in div (created by NavigatorComponent)
    in NavigatorComponent (created by Connect(NavigatorComponent))
    in Connect(NavigatorComponent) (created by RootComponent)
    in ThemeProvider (created by RootComponent)
    in StyleSheetManager (created by RootComponent)
    in RootComponent (created by Connect(RootComponent))
    in Connect(RootComponent)
    in ErrorBoundary
    in Provider

**Comment by laurent22** [comment]
Do you use a plugin or something like this? Because "toggleSidebar" indeed doesn't exist, but "toggleSideBar" does. I can't find any reference to "toggleSidebar" lowercase in the code so that's why I'm wondering if there's something custom to your profile.

**Comment by laurent22** [comment]
Ok I think I found the issue, it will be fixed in the next release.

**Comment by laurent22** [comment]
Please give a try to this version, and let me know if it works: https://github.com/laurent22/joplin/releases/tag/v1.3.18

**Comment by noteb00k** [comment]
Ouch, I didn't know github doesn't notify you when someone replies to your thread, my bad.

I downloaded the new release and now it seems to work! Thank you very much for solving it so quickly.

I noticed something interesting about toggleSidebar. When I went to modify my shortcuts, there were two toggle sidebars:

![image](https://user-images.githubusercontent.com/74029050/98447501-85f71080-2125-11eb-9f9f-abae814fea94.png)

I don't seem to be able to delete it neither nor it won't disappear when I change the sortcut combination of the invalid one; seems that it's going to stay that way. It doesn't seem to bother me but I supposed it was worth commenting, just in case it may help.
